h2o-algos/src/main/java/hex/rulefit/Condition.java

```diff
@@ -0,0 +1,112 @@
+package hex.rulefit;
+
+import water.Iced;
+import water.MRTask;
+import water.fvec.Chunk;
+import water.fvec.Frame;
+import water.fvec.NewChunk;
+import water.fvec.Vec;
+
+public class Condition extends Iced {
+    public enum Type {Categorical, Numerical};
+    public enum Operator {LessThan, GreaterThanOrEqual, In}
+    int featureIndex;
+    Type type;
+    public Operator operator;
+    public String featureName;
+    public boolean NAsIncluded;
+    public String languageCondition;
+    public double numTreshold;
+    public String[] languageCatTreshold;
+    public int[] catTreshold;
+
+    public Condition(int featureIndex, Type type, Operator operator, double numTreshold, String[] languageCatTreshold, int[] catTreshold, String featureName, boolean NAsIncluded) {
+        this.featureIndex = featureIndex;
+        this.type = type;
+        this.operator = operator;
+        this.featureName = featureName;
+        this.NAsIncluded = NAsIncluded;
+        this.numTreshold = numTreshold;
+        this.languageCatTreshold = languageCatTreshold;
+        this.catTreshold = catTreshold;
+        
+        this.languageCondition = constructLanguageCondition();
+    }
+    
+    String constructLanguageCondition() {
+        StringBuilder description = new StringBuilder();
+        description.append("(");
+        description.append(this.featureName);
```

Comment:
consecutive append calls are usually changed `description.append("(").append(this.featureName);`

Reply:
fixed 63d7852